Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -412,7 +412,24 @@ class BrowserViewController: UIViewController {
     /// Added to keyWindow, since it can then be utilized from any VC (e.g. settings modal).
     /// This also inits sync at app launch.
     private func initializeSyncWebView() {
-        Sync.shared.webView.alpha = 0.01
+        let sync = Sync.shared
+        
+        #if NO_SYNC
+        if sync.syncSeed == nil { return }
```

Comment:
Use `syncSeedArray` instead, as that is already `public` and also `readonly`. Don't want to expose `syncSeed` writtable.